src/pages/Edit/hooks/useCreateMaskbox.ts

```diff
@@ -39,11 +40,15 @@ export function useCreateMaskbox() {
       endTime,
       formData.sellAll,
       formData.selectedNFTIds,
-      formData.whiteList || ZERO_ADDRESS,
+      formData.merkleProof === '0x0000000000000000000000000000000000000000000000000000000000000000'
```

Comment:
Move `0x0000000000000000000000000000000000000000000000000000000000000000` to be a constant variable.